package.json

```diff
@@ -124,6 +124,7 @@
     "webpack-build-notifier": "^2.3.0"
   },
   "resolutions": {
-    "is-plain-object": "^5.0.0"
+    "is-plain-object": "^5.0.0",
+    "strip-ansi": "^6.0.1"
```

Comment:
it works on my machine without this (22 LTS) even after clearing everything but it doesn't seem to work on GHA even with node 22 🤷 

`7` onwards changes the export format

Reply:
and it works now 🤷